<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player1name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player3:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player4:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player5:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player6:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player2name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="player4name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="player3name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="player5name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="player6name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="player1computer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="player2computer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="player3computer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="player4computer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="player5computer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="player6computer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
